src/Components/Lobby.tsx

```diff
@@ -37,17 +37,19 @@ const Lobby = () => {
   // create new Document, save player (DB)
   const createNewGame = async () => {
     const newGameCode = cryptoRandomString({length: 5, type: 'distinguishable'});
-    setIsHost(true);
+    // setIsHost(true);
     // save Room Code
     gameDispatch({type: "joinRoom", value: newGameCode});
     // create new player
     const {player, dbPlayer}: PlayerFactoryType = PlayerFactory(playerName, 0)
 
+    setCurrentPlayer(player)
```

Comment:
here you can do `setPlayer(dbPlayer)` instead

Reply:
as for the player value that comes from PlayerFactory, im looking to refactor this out, since it seems redundant